src/features/cartographie/components/lieux-mediation-numerique-list/hub-modal/hub-modal.component.html

```diff
@@ -50,6 +50,16 @@ <h4 class="fw-bold">
               </ng-template>
             </span>
           </div>
+          <div class="text-center mt-4">
+            <label class="text-muted small">Qu'est ce qu'un Hub territorial ?</label>
```

Comment:
```suggestion
            <div class="text-muted small">Qu'est ce qu'un Hub territorial ?</label>
```
Les labels sont destinés aux éléments qui constituent les formulaires, comme input par exemple : https://www.w3docs.com/learn-html/html-label-tag.html